### PR TITLE
BaSP-TG-111: Employee-Signup

### PR DESCRIPTION
**Pull request by HernanCazeneuve**
- [X] Add SignUp button to sidebar (route: /employee/signup).
- [X] Add employee signup component.
- [X] Add library and setup inputs.
- [X] Create validations following BE with JOI.
- [X] Test feature.
- [X] Add Button Log in ( won't have functionality, will be updated in week 15) 

Captura

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,12 +0,0 @@
-{
```
Comment (GuidoCerioni) [inline .vscode/settings.json:1]: Why did you delete this file?

File: src/components/Layout/index.js [hunk 2]
```diff
@@ -35,6 +36,7 @@ function Layout() {
             <Route exact path="/super-admins/form" component={SuperAdminForm} />
             <Route exact path="/employees" component={Employees} />
             <Route exact path="/employees/form" component={EmployeeForm} />
+            <Route exact path="/employees/signup" component={EmployeeSignUp} />
```
Comment (GuidoCerioni) [inline src/components/Layout/index.js:39]: path should be /employee/signup

**Review by GuidoCerioni — CHANGES_REQUESTED** [review]
Nice work Hernán, just see the comments

File: src/components/Layout/index.js (hunk 2 above)
Comment (GuidoCerioni) [reply src/components/Layout/index.js:39]: yes, we should use employee now

**Review by CarVigil — APPROVED** [review]

**Review by tobiaspujol — APPROVED** [review]

**Pull request merged**